error_test.go

```diff
@@ -72,3 +77,38 @@ func TestIsRequestFailed(t *testing.T) {
 	assert.True(t, IsRequestFailed(ClientError{errType: RequestFailed, msg: "fail"}))
 	assert.False(t, IsRequestFailed(ClientError{errType: ClientErrorType(-1), msg: "fail"}))
 }
+
+type ErrorSuite struct {
+	ClientBaseSuite
+}
+
+func Test_Client(t *testing.T) {
+	suite.Run(t, new(ErrorSuite))
+}
+
+func (s *ErrorSuite) Test_ApiError() {
+	body := []byte(`{"httpStatus" : 401, "messages" : [ {"errorCode" : "315", "message" : "Unauthorized" } ] }`)
+	rsp := makeJsonResponse(http.StatusUnauthorized, body)
+	httpmock.RegisterResponder(http.MethodGet, "=~/datacenters", httpmock.ResponderFromResponse(rsp))
+	_, err := s.c.ListDatacenters()
+	s.Error(err)
+	s.True(IsStatusUnauthorized(err))
+	s.False(IsStatusAccepted(err))
```

Comment:
I understand calling `s.True(IsStatusUnauthorized(err))` here. But I don't understand calling `s.False(IsStatusAccepted(err))`. I mean, this test case is not supposed to test `IsStatusAccepted`.